src/instructlab/lab.py

```diff
@@ -1111,17 +1137,20 @@ def train(
         gguf_models_dir = "./models"
         if not os.path.isdir(gguf_models_dir):
             os.mkdir(gguf_models_dir)
-        shutil.copy(final_results_dir + "/ggml-model-f16.gguf", gguf_models_dir)
+        gguf_model = os.path.join(final_results_dir, "ggml-model-f16.gguf")
+        gguf_model_q = "./models/ggml-model-Q4_K_M.gguf"
+        if not skip_quantize:
+            run_quantize(gguf_model, gguf_model_q, "Q4_K_M")
+            # return gguf_model_q
+        # shutil.copy(final_results_dir + "/ggml-model-f16.gguf", gguf_models_dir)
         # cleanup original copy of model
-        os.remove(final_results_dir + "/ggml-model-f16.gguf")
+        # os.remove(final_results_dir + "/ggml-model-f16.gguf")
```

Comment:
Commented out lines needed? If not let's remove them

Reply:
fixed this up a bit, thanks for calling it out